contrib/platform/src/com/sun/jna/platform/win32/Advapi32.java

```diff
@@ -1902,4 +1976,311 @@ public int WriteEncryptedFileRaw(FE_IMPORT_FUNC pfImportCallback,
+	 *            start an application, and specify its initial drive and
+	 *            working directory.<br>
+	 * @param lpStartupInfo
+	 *            [in]<br>
+	 *            A pointer to a STARTUPINFO or STARTUPINFOEX structure. <br>
+	 *            The application must add permission for the specified user
+	 *            account to the specified window station and desktop, even for
+	 *            WinSta0\Default.<br>
+	 *            If the lpDesktop member is NULL or an empty string, the new
+	 *            process inherits the desktop and window station of its parent
+	 *            process.<br>
+	 *            The application must add permission for the specified user
+	 *            account to the inherited window station and desktop.<br>
+	 *            Windows XP: CreateProcessWithLogonW adds permission for the
+	 *            specified user account to the inherited window station and
+	 *            desktop.<br>
+	 *            Handles in STARTUPINFO or STARTUPINFOEX must be closed with
+	 *            CloseHandle when they are no longer needed.<br>
+	 *            Important If the dwFlags member of the STARTUPINFO structure
+	 *            specifies STARTF_USESTDHANDLES, the standard handle fields are
+	 *            copied unchanged to the child process without validation.<br>
+	 *            The caller is responsible for ensuring that these fields
+	 *            contain valid handle values. Incorrect values can cause the
+	 *            child process to misbehave or crash.<br>
+	 *            Use the Application Verifier runtime verification tool to
+	 *            detect invalid handles.
+	 * @param lpProcessInfo
+	 *            [out]<br>
+	 *            A pointer to a PROCESS_INFORMATION structure that receives
+	 *            identification information for the new process, including a
+	 *            handle to the process.<br>
+	 *            Handles in PROCESS_INFORMATION must be closed with the
+	 *            CloseHandle function when they are not needed.<br>
+	 * @return If the function succeeds, the return value is nonzero.<br>
+	 *         If the function fails, the return value is 0 (zero).<br>
+	 *         To get extended error information, call GetLastError.<br>
+	 *         Note that the function returns before the process has finished
+	 *         initialization.<br>
+	 *         If a required DLL cannot be located or fails to initialize, the
+	 *         process is terminated.<br>
+	 *         To get the termination status of a process, call
+	 *         GetExitCodeProcess.
+	 * @see MSDN {@link http://msdn.microsoft.com/en-us/library/windows/desktop/ms682431%28v=vs.85%29.aspx }
+	 */
+	boolean CreateProcessWithLogonW(WString lpUsername, WString lpDomain, WString lpPassword, int dwLogonFlags,
+			WString lpApplicationName, WString lpCommandLine, int dwCreationFlags, Pointer lpEnvironment,
+			WString lpCurrentDirectory, STARTUPINFO lpStartupInfo, PROCESS_INFORMATION lpProcessInfo);
+
```

Comment:
Try using _Java_'s _String_ type instead of _WString_ - I am pretty sure the JNA's native code knows how to convert correctly (make sure via a unit test...).